<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LearningWindow</class>
 <widget class="QMainWindow" name="LearningWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(205, 213, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushbutton_menu">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset; 
border-width: 1px; 
border-radius: 7px; 
border-color: black; 
padding: 4px; </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset; 
background-color: rgb(96, 138, 228);
border-width: 1px; 
border-radius: 7px; 
border-color: black; 
padding: 4px; 
font: 87 16pt 'Segoe UI Grey'; \n </string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset; 
border-width: 1px; 
border-radius: 7px; 
border-color: black; 
padding: 4px; 
background-color: rgb(96, 138, 228);
font: 87 16pt 'Segoe UI Grey'; \n </string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset; 
border-width: 1px; 
border-radius: 7px; 
border-color: black; 
padding: 4px; 
background-color: rgb(96, 138, 228);
font: 87 16pt 'Segoe UI Grey'; \n </string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>610</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset; 
border-width: 1px; 
border-radius: 7px; 
border-color: black; 
padding: 4px; 
background-color: rgb(96, 138, 228);
font: 87 16pt 'Segoe UI Grey'; \n </string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset; 
border-width: 1px; 
border-radius: 7px; 
border-color: black; 
padding: 4px; 
background-color: rgb(96, 138, 228);
font: 87 16pt 'Segoe UI Grey'; \n </string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>310</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset; 
border-width: 1px; 
border-radius: 7px; 
border-color: black; 
padding: 4px; 
background-color: rgb(96, 138, 228);
font: 87 16pt 'Segoe UI Grey'; \n </string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>460</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset; 
border-width: 1px; 
border-radius: 7px; 
border-color: black; 
padding: 4px; 
background-color: rgb(96, 138, 228);
font: 87 16pt 'Segoe UI Grey'; \n </string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>610</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset; 
border-width: 1px; 
border-radius: 7px; 
border-color: black; 
padding: 4px; 
background-color: rgb(96, 138, 228);
font: 87 16pt 'Segoe UI Grey'; \n </string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
